changelog.md

```diff
@@ -1,3 +1,79 @@
+# CITYENGINE SDK 3.2.10210 CHANGELOG
+
+This section lists changes compared to CityEngine SDK 3.2.9903.
+
+## General Info
+* CityEngine SDK 3.2.10210 is used in CityEngine 2024.0.10210.
+
+## PRT API
+* 'prt::Object::toXML()' (including all subclasses): fixed dependency on system locale and improved performance.
+
+## PRTX API
+* 'prtx::EncodePreparator::PreparationFlags': improved 'TRIANGULATE_FACES_WITH_HOLES' Action. Before, it was possible that tiny gaps in the connectivity were created. Now the mesh stays watertight. However, in some cases there might be faces with more than 3 vertices now. 
+* 'prtx::MaterialBuilder::setString()': fixed a rare crash when setting the shader.
+
+## CGA
+* New features:
+    * Visual CGA Design (*.vcga) can now be imported. You can call its start rule and override its attribute and extension nodes in the import statement.
+    * Default extension implementation forwarding: It is now possible to reuse the default implementation of an extension rule from an imported ruleset in a new extension rule. 
+* New annotations:
+    * @MaterialFile annotation
```

Comment:
no, that's done in the CGA ref. here we just give a short overview (reduced changelog) because often there is a conenction to prt / prtx functionality.